c/busybox-1.22.0-todo/cal_false-unreach-call_true-no-overflow_true-valid-memsafety.i

```diff
@@ -3790,18 +3790,20 @@ int main()
   int argc = __VERIFIER_nondet_int();
   __VERIFIER_assume(argc >= 1 && argc <= 10000);
   char **argv=malloc((argc+1)*sizeof(char*));
+  char **mem_track=argv;
   argv[argc]=0;
   for(int i=0; i<argc; ++i)
   {
     argv[i]=malloc(11);
+    mem_track[i]=argv[i];
```

Comment:
I don't understand. If I understand correctly, ``mem_track`` is an alias of argv.
Why do you reassign the contents of ``argv`` to itself via its alias?
Did you mean to allocate a second chunk of memory to keep track of the contents of ``argv`` in case they are modified elsewhere?

Reply:
Yes, I mean to keep track of the contents of `argv`.
Second commit add allocation of separate memory instead of alias.